client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/RemoteQueryWithProtostreamAnnotationsTest.java

```diff
@@ -31,7 +31,9 @@
  *
  * @author Adrian Nistor
  */
-@Test(testName = "client.hotrod.query.RemoteQueryWithProtostreamAnnotationsTest", groups = "functional")
+// TODO Hibernate Search 6 cannot handle more than one change of serialization context
```

Comment:
@gustavonalle now that lifecycle has been changed so that all the protobuf entities are in the serialization context when the cache is starting, we don't need the skip and this comment anymore. Removing it now the test passes, so I'm removing it.